frontend/src/app/pages/exercises/exercise/shared/exercise-map/feature-managers/transfer-point-feature-manager.ts

```diff
@@ -80,8 +77,8 @@ export class TransferPointFeatureManager extends ElementFeatureManager<TransferP
             return false;
         }
         if (
-            droppedElement.type !== 'vehicle' &&
-            droppedElement.type !== 'personnel'
+            droppedElement.type !== 'vehicles' &&
+            droppedElement.type !== 'personnels'
```

Comment:
Why `personnels` instead of `personnel`?

Reply:
Because `personnel` is a bad variable name ^^

But you are correct. This currently prohibits a personnel from being transferred.

In retrospect, we should have chosen a different variable name or even explicitly used `personnels` instead of `personnel` for the plural (of course, documented somewhere). Having the same name for singular and plural leads to such mistakes...

```suggestion
            droppedElement.type !== 'personnel'
```